modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,257 @@
+= AS2 Connector - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Supported Migration Paths
+
+This a direct upgrade from AS2 connector 4.x to 5.0.0.
+
+== Changes in This Release
+
+* The `Async MDN` is now handled as a separate flow. The AS2 Message Send retry is not automatic. You need to add a new flow (with an endpoint) to handle async MDN.
+
+* Users can reuse their HTTP listener configuration. Doing so means several changes for you and your trading partners:
+
+** Changing ports for an AS2-receive endpoint/port since you will use the new HTTP listener's endpoint/port configuration.
+** Changes to the load-balancer, if used.
+** You need to make your trading partners aware of the new URL/port configuration that corresponds to the HTTP listener you are reusing.
+
+* Starting with this version of the AS2 connector, a broader set of ciphers are supported: `AES128_CBC`,`AES192_CBC`,`AES256_CBC`.
+Ciphers such as: `AES128`, `AES192`, `AES256` that were used in earlier versions of the AS2 connector now map to the expanded set of these ciphers.
+
+* The namespace has changed from `as2` to `as2-mule4`. You must delete the old namespace before adding the new one.
+
+* The single global configuration is now divided into operation/source specific global configurations for a "receive endpoint" from a "send endpoint" since they are mutually exclusive.
+
+For example:
+
+** `send-config` now used by `send-with-sync-mdn` and `send-with-async-mdn` operations.
+** `listener-config` now used by `as2-listener` source.
+** `mdn-listener-config` now used by `as2-mdn-listener` source.
+
+
+== Requirements and Limitations
+
+There are some requirements and limitations you should be aware of before you upgrade to the latest version.
+
+=== Requirements
+
+Ensure you meet the following requirements before you migrate.
+
+Mule 4.1.0 or later
+
+Java 8
+
+=== Limitations
+
+Before you perform the migration, be aware of the following known limitations:
+
+* Signed copies of the document for nonrepudiation are not saved.
+
+* Authentication over HTTP is not supported.
```

Comment:
The present as2 v4 does not support Authentication over HTTP. So this limitation need not be a upgrade/migration concern.
```suggestion
```